src/test/java/cosminneagu/teorie/demoqa/BrokenLinksImages.java

```diff
@@ -0,0 +1,30 @@
+package cosminneagu.teorie.demoqa;
+
+import cosminneagu.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BrokenLinksImages {
+    public static void main(String[] args) {
+        goToTheBrokenLinksImages();
+
+    }
+
+    public static void goToTheBrokenLinksImages(){
+
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+
+        driver.get("https://demoqa.com/broken");
+
+        WebElement BrokenLinksImagesSection = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[7]/span"));
```

Comment:
Use relative xpaths instead of absolute xpaths. Applicable for all classes. Thanks!